File: URI_1009/URI_1009.cs
using System;
namespace URI1009
{
    internal static class URI
    {
        private static void Main(string[] args)
        {
            string sellerName;
            double salary, bonus, totalSalary;

            sellerName = Console.ReadLine();
            salary = Convert.ToDouble(Console.ReadLine());
            bonus = Convert.ToDouble(Console.ReadLine());
            bonus = bonus * 15 / 100;
            totalSalary = (salary + bonus);

            Console.WriteLine("TOTAL = R$ {0:F2}", totalSalary);
        }
    }
}

File: URI_1009/URI_1009.cpp
#include <iostream>
#include <string>
#include <iomanip>
using namespace std;
int main()
{
    string sellerName;
    float fixedSalary, monthlySale, bonus, totalSalary;
    cin >> sellerName >> fixedSalary >> monthlySale;
    bonus = monthlySale * 15 / 100;
    totalSalary = fixedSalary + bonus;
    cout << "TOTAL = R$ " << fixed << setprecision(2) << totalSalary << endl;
}